.travis.yml

```diff
@@ -30,13 +30,14 @@ install:
 
 script:
   # First run unit tests and bring containers down then
-  - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test up -d db influx redis
+  - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test up -d db influx redis static_tester
   - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test run --rm listenbrainz
                 dockerize
                 -wait tcp://db:5432 -timeout 60s
                 -wait tcp://influx:8086 -timeout 60s
                 -wait tcp://redis:6379 -timeout 60s
-                py.test
+                "py.test"
+  - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test run --rm static_tester npm test
```

Comment:
I would prefer it if the frontend tests were run either after all the python tests were done and containers brought down OR before any containers for the python tests are started. Right now it sits between the python unit tests and integration tests which is confusing when reading the logs.